packages/grid/stories/grid-sortColumns.stories.tsx

```diff
@@ -0,0 +1,159 @@
+import {
+  CellEditor,
+  DropdownCellEditor,
+  Grid,
+  GridColumn,
+  SortOrder,
+} from "../src";
+import "./grid.stories.css";
+import { createDummyInvestors, Investor, investorKeyGetter } from "./dummyData";
+import { useState } from "react";
+import { Scrim, ContentStatus } from "@salt-ds/lab";
+import {
+  useQuery,
+  QueryClient,
+  QueryClientProvider,
+} from "@tanstack/react-query";
+
+export default {
+  title: "Data Grid/Data Grid",
+  component: Grid,
+  argTypes: {},
+};
+
+const dummyInvestors = createDummyInvestors();
+
+const api = (sortOrder: SortOrder): Promise<Investor[]> =>
+  new Promise((resolve) => {
+    setTimeout(() => {
+      if (sortOrder === SortOrder.ASC) {
+        const sortedItems = [...dummyInvestors].sort((a, b) => {
+          return a.amount - b.amount;
+        });
+        return resolve(sortedItems);
+      }
+      if (sortOrder === SortOrder.DESC) {
+        const sortedItems = [...dummyInvestors].sort((a, b) => {
+          return b.amount - a.amount;
+        });
+        return resolve(sortedItems);
+      }
+      return resolve([...dummyInvestors]);
+    }, 1000);
+  });
+
+const queryClient = new QueryClient();
+
+export function SortColumns() {
+  return (
+    <QueryClientProvider client={queryClient}>
+      <SortColumnsImpl />
+    </QueryClientProvider>
+  );
+}
+
+function SortColumnsImpl() {
+  const [sortOrder, setSortOrder] = useState<SortOrder>(SortOrder.NONE);
+
+  const { isLoading, data, isFetching, isFetchedAfterMount } = useQuery<
+    Investor[]
+  >(["sortOrder", sortOrder], () => api(sortOrder), {
+    keepPreviousData: true,
+  });
+
+  if (isLoading) return <div>loading...</div>;
+
+  const scoreOptions = ["-", "5%", "10%", "15%", "20%"];
+
+  const onScoreChange = (row: Investor, rowIndex: number, value: string) => {
+    dummyInvestors[rowIndex].score = value;
+  };
+
+  return (
+    <div
+      style={{
+        position: "relative",
+        // @ts-ignore
+        "--salt-overlayable-background":
+          "var(--salt-color-gray-20-fade-background-readonly)",
```

Comment:
better to use palette token `--salt-palette-neutral-secondary-background-readonly`

Reply:
as discussed with Darrin, we'll use for each mode these values for the time being until secondary scrims are worked on
light mode: `--salt-color-gray-60-fade-background`
dark mode: `--salt-color-gray-300-fade-background`